src/main/java/Task.java

```diff
@@ -0,0 +1,28 @@
+public class Task {
```

Comment:
It would help to make the `Task` class abstract. `Todo`, `Deadline` and `Event` all inherit from `Task`, but we would never explicitly want to initialise a `Task` object.